xchange-okex/src/main/java/org/knowm/xchange/okex/v5/OkexExchange.java

```diff
@@ -68,6 +68,11 @@ public ExchangeSpecification getDefaultExchangeSpecification() {
     exchangeSpecification.setExchangeName("Okex");
     exchangeSpecification.setExchangeDescription("Okex Exchange");
 
+    ExchangeSpecification.ResilienceSpecification resilienceSpec = new ExchangeSpecification.ResilienceSpecification();
+    resilienceSpec.setRetryEnabled(true);
+    resilienceSpec.setRateLimiterEnabled(true);
```

Comment:
this isn't how you should do this, this is a parameter the user turns on/off when passing in the exchange spec, it shouldn't be defaulted to true

Reply:
Fixed